src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,26 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
+    private int bothside;
```

Comment:
The variable name 'bothside' is not descriptive enough. Consider renaming it to something more specific to its purpose, such as 'base' or 'baseLength', to better reflect that it represents the base of the right triangle.